css/style.css

```diff
@@ -61,19 +59,213 @@ body {
 
 /* Center the container */
 .container {
-  width: 960px;
-  margin: auto;
-  height: 500px;
+  width: 100%;
+  margin: 0 auto;
+  height: 600px;
+  background: url(../images/header_bg.jpg) no-repeat center top rgb(134,141,157);
+  background-size: cover;
+  float:left;
 }
 
 header {
+  height:600px;
+  margin:0px auto;
+  padding-top:50px;
+  padding-left:40px;
+}
+
+.tagline {
+  padding-left:-40px;
+}
+
+
+.button {
```

Comment:
@jonpackles  you need  to set a width on the container holding your link. :hammer: